pkg/kubelet/prober/results/results_manager_test.go

```diff
@@ -97,3 +97,65 @@ func TestUpdates(t *testing.T) {
 	m.Set(barID, Success, pod)
 	expectUpdate(Update{barID, Success, pod.UID}, "changed bar")
 }
+
+func TestResult_ToPrometheusType(t *testing.T) {
+	tests := []struct {
+		name     string
+		r        Result
+		expected float64
+	}{
+		{
+			name:     "result is 0",
+			r:        0,
```

Comment:
I'd suggest to use constants here and in the name, e.g. name: "result is Success", r: Success, etc.